src/constants/items.js

```diff
@@ -72,20 +78,34 @@ export const ITEMS = {
         // AHRIMSTAFF: ["AHRIMSTAFF", 100000, "SPECIAL"],
         // TRIDENT: ["TRIDENT", 150000, "SPECIAL"]
     },
-    // Tools [name, base_cost, skill]
-    TOOLS: {
-        AXE: {
+    // Tool [name, base_cost, skill]
+    Tool: {
+        Axe: {
             name: "Axe",
             cost: 100,
-            material: "SMITHINGMATS",
-            skill: "woodcutting"
+            material: "ORE",
+            skill: "woodcutting",
+            examineText: "A woodcutter's axe"
         },
-        PICKAXE: {
+        Pickaxe: {
             name: "Pickaxe",
             cost: 100,
-            material: "SMITHINGMATS",
-            skill: "mining"
+            material: "ORE",
+            skill: "mining",
+            examineText: "Used for mining"
         }
     },
-    CONSUMABLES: {}
+    Logs: {
+    	name: "Logs",
+    	cost: 5,
+    	material: "WOOD",
+    	examineText: "A number of wooden logs"
+    },
+    Bones: {
```

Comment:
I'm pretty sure I took it off the wiki for bones (then ran it through an online converter because they love using webp instead of png), but I'll double check when I get home.

Reply:
Looks like the wiki page uses big bones in the larger image but normal bones in the smaller one for some reason.